frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/EditColumnElement/EditColumnElement.tsx

```diff
@@ -55,6 +55,10 @@ export const EditColumnElement = ({
     (comp) => comp.textResourceBindings?.title && comp.dataModelBindings?.simpleBinding,
   );
 
+  const getSelectedComponentId = components.find(
+    (comp) => comp.textResourceBindings?.title === tableColumn.headerContent,
+  )?.id;
```

Comment:
I was initially thinking along the same lines as you to identify the component. However, as @mlqn pointed out for me earlier, using the title has some potential flaws—for example, if multiple components share the same title. Additionally, the new feature from #13980, which allows users to customize the table column title (`tableColumn.headerContent`), will break this logic since it separates the column title from the component's title. Given this, we might need a different approach to ensure correct component identification